pkgs/applications/window-managers/awesome/git.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, awesome, fetchFromGitHub, ... }:
+
+let
+  version = "3.6-git-g4636b11";  # "git describe: v3.5.2-398-g4636b11"
```

Comment:
Thanks for your feedback.

I've mainly done this as an exercise in nix packaging, and then thought it might be useful.  I am not using it myself, but build/install it from git anyway.

The problem with "git describe" is that awesome does not (fake-)merge the stable branch into master, so "git describe" looks rather "old", compared to the stable release/tag (currently 3.5.6).